### ISSUE TITLE
Image for "Basic plots - plot" should hint at markers

### ISSUE THREAD

**Issue by timhoffm** [matplotlib/cheatsheets#16]
Currently the image is just sine line, which could trick users in thinking that `plot` is for lines and `scatter` is for markers.

![grafik](https://user-images.githubusercontent.com/2836374/86639818-dde80400-bfd9-11ea-92df-39921907cf84.png)

I propose to additionally show another set of values with markers, e.g. something like:

![grafik](https://user-images.githubusercontent.com/2836374/86639448-32d74a80-bfd9-11ea-9571-ee4f75b37114.png)

**Comment by rougier** [comment]
You're right. I think it might be nicer to add markers along the sine with marker and markevery. Can you make a PR?

**Comment by Raghibshams456** [comment]
`import matplotlib.pyplot as plt
import numpy as np
np.random.seed(19680801)
x = np.arange(0,4*np.pi,0.1)   # start,stop,step
y = np.sin(x)
X=[1,2,3,4,5,6]
Y=[0,6,2,4,6,9]
plt.scatter(X, Y, c='orange')
plt.plot(x, y, c= "orange")
plt.gca().invert_yaxis()
plt.show()`

will this code will work??

**Comment by Raghibshams456** [comment]
![mat](https://user-images.githubusercontent.com/48047085/86816843-f3286580-c0a1-11ea-84e7-ff03c1096b9b.jpg)

**Comment by timhoffm** [comment]
> I think it might be nicer to add markers along the sine with marker and markevery.

IMHO it's important to show independent lines and markers.

**Comment by story645** [comment]
Somewhere on the cheatsheet it's mentioned that plot is a better option for very specific types of scatter situations and I think it's OK to leave this as general case plot is for lines and scatter is for points.

**Comment by Raghibshams456** [comment]
Can anybody tell me my code will work or not ...as I new to open source contribution so I am understanding things slowly....

**Comment by jklymak** [comment]
> as general case plot is for lines and scatter is for points.

 `scatter` is for when you need a third dimension, either represented as marker size or color.  Otherwise, you should just use `plot`.

**Comment by story645** [comment]
> scatter is for when you need a third dimension, either represented as marker size or color. Otherwise, you should just use plot.

for smaller datasets the difference is negligible & so you may as well not overload a function with two semantically distinct (continuous versus discrete) use cases. 

@Raghibshams456 can you be more specific? How is your code not working?

**Comment by timhoffm** [comment]
> for smaller datasets the difference is negligible & so you may as well not overload a function with two semantically distinct 

I argue the other way round: From the data semantics:

- `plot()` is primarily for functional relationship y = f(x) so that there is only one y per x. Whether you draw this with single markers or lines or both is a stylistic choice.
- `scatter()` is for drawing points in the (x, y) plane, and optionally adding additional information like color and size.
